syft/execution/plan.py

```diff
@@ -309,7 +312,7 @@ def run(self, args_: Tuple, result_ids: List[Union[str, int]]):
         # TODO: can we reuse result_ids?
         return self.__call__(*args_)
 
-    def send(self, *locations: AbstractWorker, force=False) -> PointerPlan:
+    def send(self, *locations: AbstractWorker) -> PointerPlan:
```

Comment:
One less Python-specific code path. 🎉

Reply:
Maybe I was too eager to remove that but I don't think it'll ever be useful

Reply:
I am here for it. 👍